Taskfile.yml

```
@@ -182,16 +175,30 @@ tasks:
     - CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X {{.CONTROLLER_PACKAGE}}/pkg/version.BuildVersion={{.VERSION}}" -o ./bin/{{.CONTROLLER_APP}}
 
   controller:docker-build:
-    desc: Generated the {{.CONTROLLER_APP}} Docker file.
+    desc: Builds the {{.CONTROLLER_APP}} Docker image.
     dir: "{{.CONTROLLER_ROOT}}"
     deps: [controller:build]
     sources:
     - Dockerfile
     - ./bin/{{.CONTROLLER_APP}}
     cmds:
     - docker build . -t {{.CONTROLLER_DOCKER_IMAGE}}
+
+  controller:docker-build-and-save:
+    desc: Generates the {{.CONTROLLER_APP}} Docker file.
```

minor: This is sorta confusingly phrased? A dockerfile is the input to create a docker image, yes? maybe:
```suggestion
    desc: Builds the {{.CONTROLLER_APP}} Docker image and saves it using docker save.
```